### ISSUE TITLE
HP:0002297 Red hair subclass of skin rash?

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/human-phenotype-ontology#5587]
HP:0002297 Red hair 
Subclass of:
    Skin rash
    Abnormality of hair pigmentation

The Skin rash parent was likely not intentional... and may be due to the axiom
Equivalent to:
    has part some (
        red and
        inheres in some strand of hair and
        has modifier some abnormal)
 
Skin rash is 
Equivalent to:
    has part some (
        red and
        inheres in part of some skin of body and
        has modifier some abnormal)

Is this an issue for Uberon? Maybe simply make red hair exclude subclass of skin rash?
Redheads will appreciate ;-)

**Comment by pnrobinson** [comment]
This is just in the inferred hierarchy. "strand of hair" is part-of "skin epidermis" part-of "skin of body" in UBERON.
One possible solution would be to define
```
Skin rash is
Equivalent to:
has part some (
  red and
  inheres in part of some zone of skin and
  has modifier some abnormal)
```

This is true because a rash is defined to affect a part of the body.
Another solution, probably good anyway, is to create a PATO term called "erythematous", which roughly means "red because of increase blood circulation in a defined area" 

@drseb @matentzn @LCCarmody @nicolevasilevsky 


^ http://purl.obolibrary.org/obo/UBERON_0000014

**Comment by matentzn** [comment]
I thin peters suggestion to define erythematous in PATO is the exact right way to go. Simply saying "red" in relation to a rash seems underspecified. (As I side-note I recommend to generally be very conservative with inheres in part of - even if some clinical accuracy gets lost.)

**Comment by pnrobinson** [comment]
@matentzn @drseb  can we implement as above? @LCCarmody should we make a NTR in PATO for this?

**Comment by matentzn** [comment]
Or @nicolevasilevsky she can just to the PR straight away!

**Comment by LCCarmody** [comment]
Whichever. I think @nicolevasilevsky is a PATO editor, so it would be quicker.

**Comment by nicolevasilevsky** [comment]
I can do this shortly!

**Comment by nicolevasilevsky** [comment]
@pnrobinson Is 'red' the appropriate parent?

**Comment by matentzn** [comment]
You should ask this to david or Chris probably because such decisions can have many consequences!

**Comment by nicolevasilevsky** [comment]
Thanks @matentzn, will do!

cc @cmungall and @dosumis

**Comment by pnrobinson** [comment]
@nicolevasilevsky  -- I guess there are many different ways of appearing red. Red skin is related to increased blood circulation and Red hair is related to pheomelanin. So, if PATO is modelling "redness" from a biological point of view, the answer is probably no, they should not be in a parent-child relation.

**Comment by cmungall** [comment]
agree 100% with Peter.

I think erythematous should go under morphology, but I am not aware of current discussions on what morphology is and is not

**Comment by matentzn** [comment]
morphology is fine.

**Comment by nicolevasilevsky** [comment]
thank you @cmungall and @matentzn I updated the PR.

**Comment by pnrobinson** [comment]
@nicolevasilevsky nicole, are you talking about a PR to PATO? I do not see anything here.

**Comment by matentzn** [comment]
She was talking about this: https://github.com/pato-ontology/pato/pull/363/files

**Comment by pnrobinson** [comment]
Can somebody else approve the PATO PR -- the entire tracker needs some TLC by the way, there are loads of open PRs.

**Comment by matentzn** [comment]
@dosumis is doing a lot of changes on there right now - he will merge these soon. I pinged him about the PR and he will merge soon!

**Comment by dosumis** [comment]
PATO PR merged.  Most open PRs on there are from active work on neuronal morphology terms.  Will me merged down soon.

**Comment by pnrobinson** [comment]
@dosumis is this closeable?

**Comment by matentzn** [comment]
I think what @dosumis was talking about was the PATO PRs.. You still need to change your EQ from `red` to `erythematous`. I can do it for you if you want.

**Comment by pnrobinson** [comment]
No! red hair is not erythematous (which means red because of increased blood flow). I think the current definition is fine and I am not sure if there is anything left to do here.

**Comment by drseb** [comment]
I was just thinking if we could express that `(erythematous and II some skin) equiv (red and II some skin)` @matentzn

**Comment by matentzn** [comment]
@pnrobinson I meant the skin rash :) 

@drseb  they are not equivalent though - erythematous would be a sub quality. Imagine someone spilling red ink on the skin :D 

@pnrobinson as long as your bad inference is gone (red hair subclass of red skin rash) when you use ELK, I think this can be closed!

**Comment by pnrobinson** [comment]
@matentzn 
So this definition for Skin rash is wrong
```
'has part' some 
    (red
     and ('inheres in part of' some 'skin of body')
     and ('has modifier' some abnormal))
```
we need to import erythematous from PATO
https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0040048

**Comment by pnrobinson** [comment]
This is still not quite right because hair is not part of epidermis, it is attached to epidermis

![image](https://user-images.githubusercontent.com/4943420/116726220-fc781b00-a9b0-11eb-9db6-6af89ab26357.png)


However, this probably requires an issue at uberon?

**Comment by nicolevasilevsky** [comment]
yes- this should be reported to Uberon

**Comment by pnrobinson** [comment]
Looking further, actually uberon is correct (my bad).
I think this is because we use inheres in part of 
In any case, it is not a major mistake/issue, and I think we probably cannot easily do better. I will close this, we have solved the major problem.
